### PR TITLE
Feature-TG-45: add and edit task

### PR DESCRIPTION
**Pull request by Gearyandres94**

### CODE REVIEW

File: src/Components/Tasks/Modal/Modal.module.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  width: 250px;
+  height: 170px;
+  background: white;
+  color: white;
+  z-index: 10;
+  border-radius: 16px;
+  box-shadow: 0 5px 20px 0 rgba(0, 0, 0, 0.04);
+}
+
+.modalHeader {
```
Comment (laura-brussa12) [inline src/Components/Tasks/Modal/Modal.module.css:29]: Acá no deberías aplicar camelCase sino por ejemplo **modal-header**

File: src/Components/Tasks/index.js [hunk 2]
```diff
@@ -1,9 +1,39 @@
...
 function Tasks() {
+  const [list, setList] = useState([]);
+  const [isOpen, setIsOpen] = useState(false);
+  const [idTask, setIdTask] = useState('');
+
+  useEffect(async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/tasks`);
+      const responseJson = await response.json();
+      // console.log(responseJson.data);
```
Comment (laura-brussa12) [inline src/Components/Tasks/index.js:15]: Habría que eliminar este comentario

File: src/Components/Tasks/List/List/index.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ListItem from '../ListItem/Index';
+import styles from './List.module.css';
+
+const List = ({ list, deleteItem }) => {
+  // console.log("array list", list);
```
Comment (Facund0Fernandez) [inline src/Components/Tasks/List/List/index.jsx:6]: Remember to delete unnecesary comments.

File: src/Components/Tasks/List/ListItem/Index.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styles from './listItem.module.css';
+
+const ListItem = ({ listItem, deleteItem }) => {
+  // console.log('props', listItem);
```
Comment (Facund0Fernandez) [inline src/Components/Tasks/List/ListItem/Index.jsx:5]: Comments here too.

File: src/Components/Task/index.jsx [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+          setWorkedHoursValue(response.data.workedHours);
+          setProjectNameValue(response.data.projectId.name);
+          setProjectId(response.data.projectId._id);
+        })
+        .catch((error) => {
+          console.log(error);
+        })
+        .finally(() => setLoading(false));
+    }
+    console.log(dateValue);
```
Comment (HernanCazeneuve) [inline src/Components/Task/index.jsx:41]: If not necessary could delete this console log

**Review by AlejandroSantini — APPROVED** [review]
For the future, the PR could be separated for each ticket, for better readability. Other than that, I don't see anything weird.

**Review by vadalamati — APPROVED** [review]
Good job!!

**Review by HernanCazeneuve — APPROVED** [review]

**Pull request merged**